python/federatedml/components/components.py

```diff
@@ -0,0 +1,302 @@
+#
+#  Copyright 2019 The FATE Authors. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+import importlib
+import inspect
+import typing
+from pathlib import Path
+
+from fate_arch.common import log
+from federatedml.model_base import ModelBase
+from federatedml.param.base_param import BaseParam
+
+_ml_base = Path(__file__).resolve().parent.parent.parent
+
+LOGGER = log.getLogger()
```

Comment:
change to such as “from federatedml.util import LOGGER”

Reply:
fixed